chain/stmgr/stmgr.go

```diff
@@ -300,7 +290,11 @@ func (sm *StateManager) ApplyBlocks(ctx context.Context, parentEpoch abi.ChainEp
 		return cid.Cid{}, cid.Cid{}, err
 	}
 
-	rectarr := adt.MakeEmptyArray(sm.cs.Store(ctx))
+	// XXX: Is the height correct? Or should it be epoch-1?
```

Comment:
This is matching how the VM GetNtwkVersion works, so most likely correct

(differently something to manually verify when we test the fork)